cluster-autoscaler/cloudprovider/clusterapi/README.md

```diff
@@ -254,6 +254,7 @@ rules:
     verbs:
     - get
     - list
+    - watch
```

Comment:
@dlipovetsky If you add this, then you have to update (add `watch` verb) at line no.:242 such as:

> need permission to `get`, `list`, and `watch` the infrastructure machine templates for

Reply:
Thanks, I missed that. Fixed and force-pushed.